classes/03class/exercises/c03-aws02/alanlima/README.md

```diff
@@ -0,0 +1,197 @@
+# c03-aws02
+
+## Command Execution Output
+
+- Command used to create the Application Load Balancer (ALB)
+
+    * **Create Load Balancer**
+    
+    ```bash
+    $ aws elbv2 create-load-balancer \
+        --name devopsacademy-lb \
+        --scheme internet-facing \
+        --subnets subnet-0f82d6e49b9a6dafd subnet-0299eb6eb289cef54 \
+        --security-groups sg-098dda5e94f8a3855 \
+        --tags Key=DevopsClass,Value=c03 \
+        --type application \
+        --ip-address-type ipv4
+    ```
+
+    ```yaml
+    LoadBalancers:
+        - AvailabilityZones:
+        - LoadBalancerAddresses: []
+            SubnetId: subnet-0299eb6eb289cef54
+            ZoneName: ap-southeast-2b
+        - LoadBalancerAddresses: []
+            SubnetId: subnet-0f82d6e49b9a6dafd
+            ZoneName: ap-southeast-2a
+        CanonicalHostedZoneId: Z1GM3OXH4ZPM65
+        CreatedTime: '2020-07-09T06:06:10.500000+00:00'
+        DNSName: devopsacademy-lb-1691285263.ap-southeast-2.elb.amazonaws.com
+        IpAddressType: ipv4
+        LoadBalancerArn: arn:aws:elasticloadbalancing:ap-southeast-2:097922957316:loadbalancer/app/devopsacademy-lb/8514943c4cd7b139
+        LoadBalancerName: devopsacademy-lb
+        Scheme: internet-facing
+        SecurityGroups:
+        - sg-098dda5e94f8a3855
+        State:
+            Code: provisioning
+        Type: application
+    ```
```

Comment:
yaml output? interesting.

Reply:
Yeah, it is less noisy than JSON so I had it set up by default in my profile.